actions/workflows/delivery_project_workflow.yaml

```diff
@@ -101,3 +101,15 @@ workflows:
                 delivery_base_api_url: <% $.delivery_service_url %>
                 status_link: <% $.status_link %>
                 skip_mover: <% $.skip_mover %>
+              on-success:
+                - check_ngi_ready_status: <% $.skip_mover = false %>
+
+            check_ngi_ready_status:
```

Comment:
Pretty much, yeah. Since the user will not see the project unless `ngi_ready` has been set to true we want to make sure that has actually happened. Also, me and @b97pla decided that only checking this wasn't enough, since it doesn't give us any more information that that is not set, so e.g. seeing that mover has failed (which is difficult already today) can not be done in this way.